os/include/tz_context.h

```diff
@@ -0,0 +1,146 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/****************************************************************************
+ * include/tz_context.h
```

Comment:
The policy of header location is as below:
1. os/include for standard headers like stdio.h which is defined in POSIX site.
2. os/include/tinyara  for tinyara specific headers
Because of that, sched.h is splitted at several folders.
In this point of view, is this proprietary location? This is the arm-specific header.